pkg/provider/consulcatalog/config.go

```diff
@@ -85,6 +102,85 @@ func (p *Provider) buildConfiguration(ctx context.Context, items []itemData) *dy
 	return provider.Merge(ctx, configurations)
 }
 
+func connectTransportName(item itemData) string {
+	return fmt.Sprintf("tls-%s-%s-%s", item.Namespace, item.Datacenter, item.Name)
+}
+
+type connectCert struct {
```

Comment:
(feel free to reword)
```suggestion
// connectCert holds our certificates as a client of the Consul Connect protocol.
type connectCert struct {
```